src/pages/audioPAge/audioPage.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import BookCover from './bookCover.jpg';
+import './style.scss';
+import audioBook from './audioBook.mp3';
+import AudioPlayer from 'react-h5-audio-player';
+import 'react-h5-audio-player/lib/styles.css';
+import { MDBContainer, MDBRow, MDBCol } from 'mdbreact';
+
+const AudioPage = () => {
+  const bookName = 'Mind without body';
+  const audio = audioBook;
+  return (
+    <MDBContainer className="mt-5   ">
+      <MDBRow>
+        <MDBCol sm="5" size="12">
+          <div className="mt-5 d-flex align-items-center justify-content-center order-last ">
+            <img src={BookCover} width="353" height="498" alt="bookCover" />
```

Comment:
i would love to see the image taken from a server and have a variable at the beginning of the function because it is come from a prop any way  or a server fetch use this link "https://via.placeholder.com/300x600" it gets us any size of image you want just change the 300 and 600 to any width and height you want and delete the import of the image

Reply:
okay sir thank you for your feedback i will follow them.

Reply:
yeah that's right try not to use both width and height together